Lib/fontTools/misc/plistlib.py

```diff
@@ -395,6 +395,19 @@ def totree(
     pretty_print=True,
     indent_level=1,
 ):
+    """Convert a value derived from a plist into an XML tree.
+
+    Args:
+        value: Any kind of value to be serialized to XML.
+        sort_keys: Whether keys of dictionaries should be sorted.
+        skipkeys: Keys which should be skipped when enumerating.
```

Comment:
This makes it sound like `skipkeys` is a collection, but it actually seems to be a boolean.

Reply:
Right. Didn't we say we would use Google docstring style? That has the type of each parameter in parentheses. We should use that wherever makes sense.

Reply:
We are using Google docstring style, but none of the examples on http://google.github.io/styleguide/pyguide.html have types in the `Args:` block, which is why I didn't add any. I'll add them where I can. And it looks like `skipkeys` does something different to what I thought, so I'll fix that.